plugins/verifier/schemavalidator/schemavalidation/schemavalidator.go

```diff
@@ -22,6 +22,7 @@ import (
 	"github.com/xeipuuv/gojsonschema"
 )
 
+// Validates content from a byte array against a schema from a URL
```

Comment:
Thanks for clarifying @mluker . Maybe we could rename to ValidateAgainstSchemaURL and ValidateAgainstOfflineSchema ?

Reply:
renaming Validate2 to ValidateAgainstOfflineSchema makes sense but Validate takes a string which can be a canonical file path or URL.

Reply:
sounds good @mluker.  Renaming Validate2 is great, contributors will be able to work out the difference from reading the method descriptions.